### ISSUE TITLE
list assignment index out of range

### ISSUE THREAD

**Issue by miwaliu** [nerox8664/onnx2keras#64]
tensorflow 2.1.0 
onnx2keras 0.0.19

used torch.cat appear list assignment index out of range

import numpy as np
import torch
import torch.nn as nn
from torch.autograd import Variable
from onnx2keras import onnx_to_keras, check_torch_keras_error
import onnx
import torch.nn.functional as F
import tensorflow as tf

def conv_bn(inp, oup, stride = 1):
    return nn.Sequential(
        nn.Conv2d(inp, oup, 3, stride, 1, bias=False),
        #nn.BatchNorm2d(oup),
        nn.ReLU(inplace=True)
    )

class Test(nn.Module):
    def __init__(self):
        super(Test, self).__init__()
        self.pool = nn.MaxPool2d(2)
        self.conv1 = conv_bn(3, 16, 2)

    def forward(self, inputs):
        x1 = self.conv1(inputs)

        #print(inputs.shape,x1.shape)
        x2 = torch.cat([x1.reshape(1,19200,4), inputs.reshape(1,14400,4)], dim=1)
        return x2

model = Test().eval().cpu()

input_np = np.random.uniform(0, 1, (1, 3, 120, 160))
input_var = torch.FloatTensor(input_np)

torch.onnx.export(model, input_var, 'test.onnx', verbose=True,
                  input_names=['input'], output_names=['output'])
onnx_model = onnx.load('test.onnx')

k_model = onnx_to_keras(onnx_model=onnx_model, input_names=['input'], change_ordering=True, verbose=False, name_policy='renumerate')

# Save model to SavedModel format
tf.saved_model.save(k_model, "./models")

any idea?

**Comment by miwaliu** [comment]
close
